### PR TITLE
Final report DFD

### PR DESCRIPTION
**Pull request by Ansper11**
This is the DFD for the final report, please check.

### CODE REVIEW

**Review by Nick-McGee — CHANGES_REQUESTED** [review]
The DFD level 1 is a bit messy and hard to follow. I recommend straightening the lines more, increasing the space between elements if needed, and making it more uniform.

**Comment by Ansper11** [comment]
Hi, I changed the DFD level 1, please check

**Comment by yyht6300a** [comment]
It makes sense. Looks good
